poetry.lock

```diff
@@ -246,7 +264,7 @@ toml = ["tomli"]
 
 [[package]]
 name = "cryptography"
-version = "37.0.4"
+version = "38.0.4"
```

Comment:
This is the cause: https://stackoverflow.com/questions/73859249/attributeerror-module-openssl-ssl-has-no-attribute-sslv3-method

Reply:
@jvdd

Reply:
Isn't this the cause https://github.com/wkeeling/selenium-wire/issues/596

Reply:
@jonasvdd

Reply:
Updating the `selenium-wire` dev dependency to `5.x` fixes the issue (as was stated in the above selenium-wire issue)